startup/gui/editor.py

```diff
@@ -132,18 +134,41 @@ def __registerNodeSetFollowsSceneSelectionItems( editor, menuDefinition ) :
 		if target is not editor :
 			__addFollowMenuItem( menuDefinition,
 				editor, target,
-				"Following Scene Selection",
+				"Following/Scene Selection",
 				DriverModeSceneSelectionSource,
 				itemNameCounts
 			)
 
+# Numeric bookmark follows
+def __registerNodeSetFollowsNumericBookmarkItems( editor, menuDefinition ) :
```

Comment:
Another thing to ponder. Have we ended up with a separation between library and config that is a little artificial here? Since CompoundEditor has code specific to _displaying_ bookmark sets and editor following, it seems strange to push the code for the _menu_ out into a config. I think what would be most consistent with our general approach would be to have something like `__registerNodeSetFollowsNumericBookmarkItems()` as public API somewhere and then just call it from the config. Again, not something we need to do now, but something to ponder...

Reply:
https://github.com/GafferHQ/gaffer/pull/3323/commits/816c798fefc20c9333428c4eb6588014dcdbe2e9 Moves the bookmark set items to be with the others.

Reply:
I looked into moving the other registrations to `CompoundEditor` (to keep it in the 'big lump of shame' until we move to `GraphComponents`), but it requires `GafferSceneUI`, so feels like keeping that in the config for now is a reasonable separation. The `CompoundEditor` will still need to know about some of the specifics the link types for serialisation, but they should be restricted to `Gaffer` not `GafferScene`.